bib/cmd/bootc-image-builder/main.go

```diff
@@ -114,6 +130,13 @@ func makeManifest(c *ManifestConfig, cacheRoot string) (manifest.OSBuildManifest
 			return nil, fmt.Errorf("failed to pull container image: %w", err)
 		}
 	}
+	cntSize, err := getContainerSize(c.Imgref)
+	if err != nil {
+		return nil, fmt.Errorf("cannot get container size: %w", err)
+	}
+	c.Filesystems = []blueprint.FilesystemCustomization{
+		{Mountpoint: "/", MinSize: cntSize * 2},
```

Comment:
I'm not fully following the logic under `disk.NewPartitionTable`; will the `/` filesystem still be grown in this scenario to fit the overall disk?

Reply:
(And if this filesystem is larger than `DEFAULT_SIZE`, does it go past `DEFAULT_SIZE`?)

Reply:
The PartitionTable.relayout comments says the following:
```
// Dynamically calculate and update the start point for each of the existing
// partitions. Adjusts the overall size of image to either the supplied
// value in `size` or to the sum of all partitions if that is larger.
// Will grow the root partition if there is any empty space.
// Returns the updated start point.
```
so unless I misread this it should DTRT *but* I will write some tests to ensure that this is really the case to ensure we do not regress here.